topics/epigenetics/tutorials/ewas-suite/workflows/ewas_training.ga

```diff
@@ -0,0 +1 @@
su.edu/repos/kpbioteam/minfi_dropsnp/minfi_dropsnp/0.1.0", "type": "tool"}, "9": {"tool_id": "testtoolshed.g2.bx.psu.edu/repos/kpbioteam/minfi_dmp/minfi_dmp/0.1.0", "tool_version": "0.1.0", "outputs": [{"type": "txt", "name": "output1"}], "workflow_outputs": [{"output_name": "output1", "uuid": "f060686d-ca40-4719-b6d1-b7f24d84cf11", "label": null}], "input_connections": {"input2": {"output_name": "output", "id": 2}, "input1": {"output_name": "output1", "id": 8}}, "tool_state": "{\"input2\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"input3\": \"\\\"categorical\\\"\", \"input1\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"input4\": \"\\\"0.5\\\"\", \"input5\": \"\\\"TRUE\\\"\", \"__page__\": null, \"__rerun_remap_job_id__\": null, \"chromInfo\": \"\\\"/home/ubuntu/galaxy/tool-data/shared/ucsc/chrom/?.len\\\"\"}", "id": 9, "tool_shed_repository": {"owner": "kpbioteam", "changeset_revision": "603a77133ae8", "name": "minfi_dmp", "tool_shed": "testtoolshed.g2.bx.psu.edu"}, "uuid": "3efb6f3b-cf9a-4f9b-a02d-33e7b0d2b171", "errors": null, "name": "minfi_dmp", "post_job_actions": {}, "label": null, "inputs": [{"name": "input2", "description": "runtime parameter for tool minfi_dmp"}, {"name": "input1", "description": "runtime parameter for tool minfi_dmp"}], "position": {"top": 312.96875, "left": 2851.25}, "annotation": "", "content_id": "testtoolshed.g2.bx.psu.edu/repos/kpbioteam/minfi_dmp/minfi_dmp/0.1.0", "type": "tool"}, "10": {"tool_id": "testtoolshed.g2.bx.psu.edu/repos/kpbioteam/minfi_dmr/minfi_dmr/0.1.0", "tool_version": "0.1.0", "outputs": [{"type": "bed", "name": "output1"}], "workflow_outputs": [{"output_name": "output1", "uuid": "cc7ec77d-e98c-4dd6-ada4-e2e9007b6baa", "label": null}], "input_connections": {"input2": {"output_name": "output", "id": 2}, "input1": {"output_name": "output1", "id": 8}}, "tool_state": "{\"input2\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"input3\": \"\\\"resistant\\\"\", \"input1\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"input6\": \"\\\"2\\\"\", \"input7\": \"\\\"0.1\\\"\", \"input4\": \"\\\"sensitive\\\"\", \"input5\": \"\\\"250\\\"\", \"__page__\": null, \"input8\": \"\\\"permutation\\\"\", \"input9\": \"\\\"TRUE\\\"\", \"chromInfo\": \"\\\"/home/ubuntu/galaxy/tool-data/shared/ucsc/chrom/?.len\\\"\", \"__rerun_remap_job_id__\": null}", "id": 10, "tool_shed_repository": {"owner": "kpbioteam", "changeset_revision": "53953bab9d1e", "name": "minfi_dmr", "tool_shed": "testtoolshed.g2.bx.psu.edu"}, "uuid": "fda81678-7844-470c-ac54-5eb449aa6149", "errors": null, "name": "minfi_dmr", "post_job_actions": {}, "label": null, "inputs": [{"name": "input2", "description": "runtime parameter for tool minfi_dmr"}, {"name": "input1", "description": "runtime parameter for tool minfi_dmr"}], "position": {"top": 546.9375, "left": 2863.25}, "annotation": "", "content_id": "testtoolshed.g2.bx.psu.edu/repos/kpbioteam/minfi_dmr/minfi_dmr/0.1.0", "type": "tool"}}, "annotation": "", "a_galaxy_workflow": "true"}
```

Comment:
Can you extract the `tools.yaml` file there?
You can run:

```
$ workflow-to-tools -w path/to/workflow -o path/to/tools.yaml
```

Doc: http://galaxyproject.github.io/training-material/topics/contributing/tutorials/create-new-tutorial-technical/tutorial.html#creating-the-toolsyaml-recommended
I can also do it if you want.
Thanks a lot!